Example/ExampleApp/SessionDetails/SessionDetailViewModel.swift

```diff
@@ -0,0 +1,140 @@
+import Foundation
+import Combine
+import WalletConnectAuth
+
+@MainActor
```

Comment:
Why does it need to be a `@MainActor`? Does it have something to do with setting a `@Published` var during an async call?

Reply:
Yes, ensure what UI update in main thread